diesel_cli/src/main.rs

```diff
@@ -191,22 +221,23 @@ fn redo_latest_migration<Conn>(conn: &Conn) where
     }).unwrap_or_else(handle_error);
 }
 
-fn handle_error<E: Error>(error: E) {
+fn handle_error<E: Error, T>(error: E) -> T {
```

Comment:
No, this function never returns. I just noticed it was being used in a strange way on some lines affected by this change, and modified the signature so it can be used with `unwrap_or_else` instead of `map_err.unwrap`